### ISSUE TITLE
Add a `press` helper?

### ISSUE THREAD

**Issue by leizhao4** [cibernox/ember-native-dom-helpers#42]
How about adding a `press` helper which fires `keydown`, `keypress` and then `keyup`?

**Comment by pixelhandler** [comment]
@leizhao4 did you notice the example in the README that shows `keyEvent` ? https://github.com/cibernox/ember-native-dom-helpers#usage

```js
keyEvent('.other-input', 'keyup', 40); // down arrow
```

**Comment by cibernox** [comment]
Actually having a single helper that simulates a full key interation (fires keydown, keyup and keypress in sequence) sounds reasonable, the same way `click` fires mousedown, focus, mouseup and click in sequence.

I'd call it `pressKey` tho, but it sounds reasonable.

**Comment by cibernox** [comment]
I created the `under consideration` label and added to this issue. To see what other think. I'm personally :+1:, but I don't feel strongly either way.

**Comment by leizhao4** [comment]
Yes that's exactly what I mean. After playing around with keyboard events (in Chrome), it looks more complex than I expected and I think there's plenty of room for a new helper.

The events seems to fire in this sequence:

1. `keyDown`.
2. `keyPress` - But not always. Arrow keys don't have this event, for example.
3. `textInput` - Similar to `keyPress` but only for certain element types.
4. `keyUp`.

**Comment by rwjblue** [comment]
I'm in favor, though I would rather see `pressKey` also.

**Comment by cibernox** [comment]
Green light then for someone to implement it.
